src/server/pachyderm_test.go

```diff
@@ -1784,11 +1784,14 @@ func TestProvenance(t *testing.T) {
 }
 
 // TestProvenance2 tests the following DAG:
-//   A
-//  / \
+//
+//	 A
```

Comment:
is this a VSCode autoformatting thing? I saw the same thing on Jose's PR the other day